docs/UserGuide.md

```diff
@@ -18,85 +18,107 @@ title: User Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 2. About
+## 2. How to use this User Guide
 
-### 2.1 Using this User Guide
+Welcome to *_MyInsuRec_ User Guide*!
 
-Welcome to *MyInsuRec User Guide*!
+Our team would like to thank you for choosing to use _MyInsuRec_! <br>
 
-This guide explains how you can use *MyInsuRec* to organize and manage your clients, meetings and products. This guide has been structured for users to quickly find and understand what they need.
+In order to smoothen the learning curve for you,
+this guide would allow you to effectively learn how to use _MyInsuRec_.
```

Comment:
this should be one line

Reply:
i believe in markdown unless it is double new line or &lt;br&gt; then it will create a new line. This should still be one line

Reply:
yes, it is one line

Reply:
yes I understand but normally one statement should be one line. I think this is mentioned in the lecture or GFMD tutorial.

Reply:
alright then i change back later